index.html

```diff
@@ -23,21 +23,32 @@
                     <path fill-rule="evenodd"
                         d="M2.5 12a.5.5 0 0 1 .5-.5h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5zm0-4a.5.5 0 0 1 .5-.5h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5zm0-4a.5.5 0 0 1 .5-.5h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5z" />
                 </svg>
+                <div class="nav-list">
+                    <ul>
+                        <li><a href="#portfolio">Portfolio</a></li>
+                        <li><a href="#about">About me</a></li>
+                        <li><a href="#contact">Contact</a></li>
+                        <li><a href="https://medium.com/BlessmanNewton"><i class="fa fa-envelope-o"
+                                    aria-hidden="true"></i></a></li>
+                    </ul>
+                </div>
             </div>
         </nav>
     </header>
-    <section>
+    <section id="portfolio">
         <div class="container">
             <div class="row">
-                <h1>Hey There. <br> I'm Bless.</h1>
-                <h2>I'm a Software Developer</h2>
-                <p class="text">
-                    I can help you build a product , feature
-                    or website Look through some of my
-                    work and experience! If you like what you
-                    see and have a project you need coded,
-                    don’t hesitate to contact me.
-                </p>
+                <div class="header-content">
+                    <h1>Hey There. <br> I'm Blessman.</h1>
+                    <h2>I'm a Software Developer</h2>
+                    <p class="text">
+                        I can help you build a product , feature
+                        or website Look through some of my
+                        work and experience! If you like what you
+                        see and have a project you need coded,
+                        don’t hesitate to contact me.
+                    </p>
```

Comment:
Good job by making some of the changes requested by the previous reviewer 👍🏼 , one small thing still needs to be fixed is the font-weight to reduce it use => `font-weight: initial;`